project/Dependencies.scala

```diff
@@ -87,81 +110,107 @@ object Dependencies {
 object Dependency {
   object V {
     // runtime deps versions
-    val Aws = "1.11.243"
-    val Alpakka  = "0.14"
-    val Chaos = "0.10.0"
-    val Guava = "20.0"
-    val Mesos = "1.5.0"
-    // Version of Mesos to use in Dockerfile.
-    val MesosDebian = "1.5.0-2.0.1"
     val Akka = "2.5.7"
     val AkkaHttp = "10.0.11"
+    val Alpakka  = "0.14"
     val ApacheCommonsCompress = "1.13"
     val ApacheCommonsIO = "2.6"
     val AsyncAwait = "0.9.7"
+    val Aws = "1.11.243"
+    val Diffson = "2.2.2"
+    val Guava = "20.0"
+    val Guice = "4.1.0"
+    val Hibernate = "5.2.1.Final"
```

Comment:
[Noooooooooooooooooo](http://www.nooooooooooooooo.com/)

Reply:
I'll see if this can be removed. It came from Chaos. I don't think use it.